### DISCUSSION TITLE
How to use ~/.ssh/config file properties in sshj client

### DISCUSSION THREAD

**Discussion by ptg1792** [hierynomus/sshj#845] [General]
Can we use the host ~/.ssh/config properties file while initiating the sshj client? We have custom properties defined in there which are not part of net.schmizz.sshj.Config class.

Is there any way, we can add custom config in key value pairs in it?

**Comment by hierynomus** [comment]
Nope, all possible config options for sshj are exposed purely through the config class. If you want to populate that from your `~/.ssh/config` you would have to write a parser yourself.
